lib/crazy_eight_game.rb

```diff
@@ -0,0 +1,161 @@
+require 'JSON'
+require 'rest-client'
+require 'pry'
+
+class CrazyEightGame < ActiveRecord::Base
+
+    def new_deck
+        self.deck_id = JSON.parse(RestClient.get("https://deckofcardsapi.com/api/deck/new/shuffle/"))['deck_id']
+        update_remaining(JSON.parse(RestClient.get("https://deckofcardsapi.com/api/deck/new/shuffle/"))['remaining'])
+        save
+    end
+
+    def draw_from_api(count)
+        response = RestClient.get("https://deckofcardsapi.com/api/deck/#{deck_id}/draw/?count=#{count}")
+        update_remaining(JSON.parse(response)['remaining'])
+        JSON.parse(response)['cards']
+    end
+
+    def deal_start_hand(player)
+        draw_from_api(7).each do |c| 
+            Hand.create(location: player, deck_id: deck_id, suit: c['suit'], value: c['value'], code: c['code'])
```

Comment:
```suggestion
            Hand.create(
                location: player, 
                deck_id: deck_id, 
                suit: card['suit'], 
                value: card['value'], 
                code: card['code']
            )
```
Nitpick: Long lines of code should also be avoided. Typically I the cut off I have seen is ~80 characters. For long assignments like this simply split them up after each assignment.